auth/lib/services/preference_service.dart

```diff
@@ -23,9 +23,11 @@ class PreferenceService {
   static const kShouldAutoFocusOnSearchBar = "should_auto_focus_on_search_bar";
   static const kShouldMinimizeOnCopy = "should_minimize_on_copy";
   static const kCompactMode = "vi.compactMode";
+  static const kAppInstallTime = "appInstallTime";
 
   Future<void> init() async {
     _prefs = await SharedPreferences.getInstance();
+    await setAppInstallTime();
```

Comment:
Let's remove this method.

int getAppInstalledTime() {
    if (_prefs.containsKey(kAppInstallTime)) {
      return _prefs.getInt(kAppInstallTime)!;
    } else {
      int installedTimeinMillis = DateTime.now().millisecondsSinceEpoch;
      _prefs.setInt(kAppInstallTime, installedTimeinMillis).ignore();
     return installedTimeinMillis;
    }
  }